### PR TITLE
Add org.kde.kalm

### PR DESCRIPTION
**Pull request by plata**
Kalm can teach you different breathing techniques.

https://apps.kde.org/kalm/

### Please confirm your submission meets all the criteria

- [x] Please describe your application briefly.
- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an author/developer/upstream contributor of the project.
- [x] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Comment by hfiguiere** [comment]
cc @flathub/kde

File: org.kde.kalm.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    sources:
+      - type: archive
+        url: https://download.kde.org/stable/release-service/24.05.0/src/kalm-24.05.0.tar.xz
+        sha256: 55999d5872e7b4569f5c31b893f19f91a2fdf0d89a8ba4e672d9b69e45a1bab6
+        #x-checker-data:
+        #  type: anitya
+        #  project-id: TODO
+        #  stable-only: true
+        #  url-template: https://download.kde.org/stable/release-service/$version/src/kalm-$version.tar.xz
+    cleanup:
```
Comment (hfiguiere) [inline org.kde.kalm.yml:38]: I'm pretty sure you want this cleanup in the top level.

**Comment by hfiguiere** [comment]
bot, build org.kde.kalm

**Comment by plata** [comment]
```
FB: Running: appstreamcli compose --no-partial-urls --prefix=/ --origin=org.kde.kalm --media-baseurl=https://dl.flathub.org/media --media-dir=/srv/buildbot/worker/build-aarch64-3/build/.flatpak-builder/rofiles/rofiles-sTrwbi/files/share/app-info/media --result-root=/srv/buildbot/worker/build-aarch64-3/build/.flatpak-builder/rofiles/rofiles-sTrwbi/files --data-dir=/srv/buildbot/worker/build-aarch64-3/build/.flatpak-builder/rofiles/rofiles-sTrwbi/files/share/app-info/xmls --icons-dir=/srv/buildbot/worker/build-aarch64-3/build/.flatpak-builder/rofiles/rofiles-sTrwbi/files/share/app-info/icons/flatpak '--components=org.kde.kalm,org.kde.kalm.desktop' /srv/buildbot/worker/build-aarch64-3/build/.flatpak-builder/rofiles/rofiles-sTrwbi/files
appstreamcli: error while loading shared libraries: libappstream.so.5: cannot open shared object file: No such file or directory
Error: ERROR: appstreamcli compose failed: Child process exited with code 127
```
What's happening there? I see the same error in other builds as well.

**Comment by bbhtt** [comment]
Looks good waiting for a comment from Flathub KDE team.

File: org.kde.kalm.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+          url-template: https://download.kde.org/stable/kirigami-addons/kirigami-addons-$version.tar.xz
+  - name: kalm
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://download.kde.org/stable/release-service/24.05.0/src/kalm-24.05.0.tar.xz
+        sha256: 55999d5872e7b4569f5c31b893f19f91a2fdf0d89a8ba4e672d9b69e45a1bab6
+        #x-checker-data:
+        #  type: anitya
+        #  project-id: TODO
```
Comment (travier) [inline org.kde.kalm.yml:42]: If this is part of KDE Apps/Gears then there is a common ID for all apps.

**Review by travier — COMMENTED** [review]
For now, we have manifests as JSON. We might change that in the future but it's what it is. There have been exception for Mobile apps.

**Comment by travier** [comment]
Otherwise manifests looks good. I've not tested the app.

File: org.kde.kalm.yml (hunk 2 above)
Comment (plata) [reply org.kde.kalm.yml:42]: It is. What's the ID?

**Comment by plata** [comment]
> For now, we have manifests as JSON. We might change that in the future but it's what it is. There have been exception for Mobile apps.

It's basically the same what I have for org.kde.telly-skout.

File: org.kde.kalm.yml (hunk 2 above)
Comment (plata) [reply org.kde.kalm.yml]: This: https://release-monitoring.org/project/8763/?

Comment (travier) [reply org.kde.kalm.yml]: yes

**Comment by travier** [comment]
bot, build org.kde.kalm

**Comment by travier** [comment]
Tested, works well, and LGTM

**Comment by travier** [comment]
Ready for merge

**Pull request closed without merging**